mxcube3/ui/reducers/shapes.js

```diff
@@ -24,6 +24,16 @@ export default (state = initialState, action) => {
 
         return { ...state, shapes };
       }
+    case 'UPDATE_SHAPES':
```

Comment:
Hey I already have this code in my master, did you do a rebase or merge with master ?

Reply:
I do not what I did! You know how much I like git...